src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -1,14 +1,110 @@
 <template>
-<div>Conditional Styles inspector view</div>
+<div>
+    <div v-if="!conditionalStyles.length"
+         class="holder c-c-button-wrapper align-left">
+        <button
+            class="c-c-button c-c-button--minor add-criteria-button"
+            @click="addConditionSet">
+            <span class="c-c-button__label">Use conditional styling</span>
+        </button>
+    </div>
+    <div v-else>
+        <div class="holder c-c-button-wrapper align-left">
+            <button
+                class="c-c-button c-c-button--minor add-criteria-button"
+                @click="removeConditionSet">
+                <span class="c-c-button__label">Remove conditional styling</span>
+            </button>
+        </div>
+        <ul>
+            <li v-for="conditionStyle in conditionalStyles"
+                :key="conditionStyle.conditionIdentifier.key">
+                <conditional-style :condition-identifier="conditionStyle.conditionIdentifier"
+                                   :condition-style="conditionStyle.style"/>
+            </li>
+        </ul>
+    </div>
+</div>
 </template>
 
 <script>
 
+import ConditionalStyle from "./ConditionalStyle.vue";
 export default {
     components: {
+        ConditionalStyle
     },
     inject: [
-        'openmct'
-    ]
+        'openmct',
+        'domainObject',
+        'layoutItem'
+    ],
+    data() {
+        return {
+            conditionalStyles: []
+        }
+    },
+    destroyed() {
+        if (this.styleRuleManager) {
+            this.styleRuleManager.destroy();
```

Comment:
I don't see any other references to `styleManager`. Is it redundant?

Reply:
Removed.